getting_started.md

```diff
@@ -319,8 +319,9 @@ Example:
 analysis-runner \
     --dataset <dataset> \
     --description "Description of the run" \
+    --access-level <level>
     --output-dir gs://<bucket> \
-    path/to/myscript.py -p 3
+    python3 path/to/myscript.py -p 3
```

Comment:
Should we also include documentation on usage that looks like 
`python3 -m path.to.myscript`?

Reply:
Good question! Python modules are complicated, but maybe we could link to a good description if you can find one?